simulation/traffic_simulator/src/behavior/action_node.cpp

```diff
@@ -237,7 +237,8 @@ boost::optional<double> ActionNode::getDistanceToTargetEntityOnCrosswalk(
 {
   if (status.lanelet_pose_valid) {
     auto polygon = hdmap_utils->getLaneletPolygon(status.lanelet_pose.lanelet_id);
-    return spline.getCollisionPointIn2D(polygon);
+    const auto s = spline.getCollisionPointIn2D(polygon, false, true);
+    return s;
```

Comment:
I think the following is better.
```
    return spline.getCollisionPointIn2D(polygon, false, true);
```
Because, RVO (Return Value Optimization) is guaranteeed, but NRVO (Named Return Value Optimization) is not guaranteed.

Reply:
Fixed in 96393bf01852830a5bbe199a841be07b13643a3b